static/app/components/notificationActions/forms/onCallServiceForm.tsx

```diff
@@ -79,6 +79,13 @@ function PagerdutyForm({
       },
     }));
   };
+  const keySelectionTxt =
+    onCallService === 'pagerduty'
+      ? t('account with the service')
+      : t('account with the key');
+
+  const dropdownTxt =
+    onCallService === 'pagerduty' ? t('Select Service') : t('Select Key');
```

Comment:
what is a key and what is a team? i notice elsewhere that you also use team to refer to an opsgenie service. it should be consistent

Reply:
I'll change this to team to be consistent